safe/definitionsv4/hazard.py

```diff
@@ -6,7 +6,7 @@
     flood_hazard_classes,
     tsunami_hazard_classes,
     ash_hazard_classes,
-    cyclone_au_bom_hazard_classes)
+    cyclone_au_bom_hazard_classes, cyclone_sshws_hazard_classes)
```

Comment:
Can you separate with newlines please.
``` 
cyclone_au_bom_hazard_classes, 
cyclone_sshws_hazard_classes)```

I'm not a fan of acronyms because nobody else will understand them, but I understand why you put there there. Perhaps at least add a comment above indicating the meaning of the acronyms...